# HG changeset patch
# Parent 7faa124a192fa5d67096119031af02d1f825c372
# User sbr
# Date 1347551772 -7200

Fix: bit number of 'data' in CmdModifyOrder comment.

diff -r 7faa124a192f src/order_cmd.cpp
--- a/src/order_cmd.cpp	Sat Sep 15 17:45:08 2012 +0000
+++ b/src/order_cmd.cpp	Sun Sep 16 11:46:09 2012 +0200
@@ -1177,7 +1177,7 @@
  *                        the maximum vehicle order id is 254.
  * @param p2 various bitstuffed elements
  *  - p2 = (bit 0 -  3) - what data to modify (@see ModifyOrderFlags)
- *  - p2 = (bit 4 - 15) - the data to modify
+ *  - p2 = (bit 4 - 14) - the data to modify
  * @param text unused
  * @return the cost of this operation or an error
  */
